docs/roadmap.md

```diff
@@ -0,0 +1,49 @@
+# Roadmap to Federated Learning on Azure ML
+
+## Contents
+Here we propose a roadmap for onboarding to Federated Learning (FL) on Azure ML. The roadmap follows the Crawl-Walk-Run<!-- -Fly--> framework, where we suggest to start with a simple example, and then progressively add more complexity.
+
```

Comment:
can we add a section on who's the audience for this tutorial? and what value they'll find in the page?

(like learning goals for different personas: what a ML/DS engineer will find interesting, what a team lead will find, etc?)

Reply:
done. please take another look